src/status_im/extensions/ethereum.cljs

```diff
@@ -27,6 +29,21 @@
  (fn [{db :db} [_ on-result message]]
    (when on-result {:dispatch (on-result {:error message :result nil})})))
 
+(defn- wrap-with-resolution [db arguments address-keyword f]
```

Comment:
what is `f` ?

Reply:
It's the function wrapped that will be called. Any better name?

Reply:
`cb`? `callback`? `eventfunction`?